packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -86,6 +86,7 @@ def _create_client(
     output_parameters_with_type = get_output_parameters_with_type(
         output_metadata, built_in_import_modules, custom_import_modules, enum_values_by_type
     )
+    custom_import_modules.sort()
```

Comment:
Why is this necessary?

Reply:
`get_output_parameters_with_type` adds custom import modules by what value it gets first from output_metadata.values() [here](https://github.com/ni/measurement-plugin-python/blob/7ea52e5a2b45da2af95e3f66e0563a41239ccfdd/packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py#L279C5-L279C46) and since the DoubleXYData value is before Double2DArray value [here](https://github.com/ni/measurement-plugin-python/blob/7ea52e5a2b45da2af95e3f66e0563a41239ccfdd/packages/generator/tests/test_assets/example_renders/measurement_plugin_client/non_streaming_data_measurement_client.py#L376C13-L385C15) it is picking that up first. But then it throws an error later down the line regarding import modules not being in correct order.

Now, on retrospecting it would be better to just add Double2DArray before DoubleXYData but this needs to be addressed that in these types of cases we need to be mindful of the import order

Reply:
Value of custom_import_module if we are not sorting - 
['from ni_measurement_plugin_sdk_service._internal.stubs.ni.protobuf.types.xydata_pb2 import DoubleXYData', 'from ni_measurement_plugin_sdk_service._internal.stubs.ni.protobuf.types.array_pb2 import Double2DArray']